modin/pandas/test/test_io.py

```diff
@@ -2090,6 +2091,51 @@ def test_ExcelFile(self, make_excel_file):
             modin_excel_file.close()
             pandas_excel_file.close()
 
+    def test_ExcelFile_bytes(self, make_excel_file):
+        unique_filename = make_excel_file()
+        with open(unique_filename, mode="rb") as f:
+            content = f.read()
+
+        modin_excel_file = pd.ExcelFile(content)
+        pandas_excel_file = pandas.ExcelFile(content)
+
+        df_equals(modin_excel_file.parse(), pandas_excel_file.parse())
+        assert isinstance(modin_excel_file, pd.ExcelFile)
```

Comment:
what is the point of this check?.. shouldn't it be `isinstance(modin_excel_file, pandas.ExcelFile)` or whatever?

Reply:
Removed